realm/realm-jni/src/io_realm_internal_Group.cpp

```diff
@@ -289,3 +289,23 @@ JNIEXPORT jboolean JNICALL Java_io_realm_internal_Group_nativeEquals(
     } CATCH_STD()
     return false;
 }
+
+JNIEXPORT jboolean JNICALL Java_io_realm_internal_Group_nativeIsEmpty(
+    JNIEnv*, jobject, jlong nativeGroupPtr)
+{
+    Group* grp = G(nativeGroupPtr);
+    const std::string table_prefix = "class_";
```

Comment:
Yes, it is declared in `io.realm.internal.Table` but apparently `javah` does not generate macros for string constants.